src/protego.py

```diff
@@ -307,8 +307,8 @@ def __init__(self):
     @classmethod
     def parse(cls, content):
         o = cls()
-        if isinstance(content, bytes):
-            content = content.decode()
+        if not isinstance(content, str):
+            raise ValueError("Not a string - cannot parse: {}".format(content))
```

Comment:
If `content` is non-UTF-8 bytes, `"{}".format(content)` will cause an issue as it will try to decode `content`.

```suggestion
            raise ValueError(f"Protego.parse expects str, got {type(content).__name__}")
```